src/hooks/useENSProfile.ts

```diff
@@ -47,5 +55,7 @@ export default function useENSProfile(
     [accountAddress, data?.owner?.address]
   );
 
-  return { data, isLoading, isOwner, isSuccess };
+  const isPrimaryName = isLowerCaseMatch(walletNames[accountAddress], name);
```

Comment:
prefer native `.toLowerCase()`? 

`isLowerCaseMatch` uses lodash, and i think we are trying to migrate away from that.

Reply:
oh